lsp/nls/src/linearization/completed.rs

```diff
@@ -168,8 +168,13 @@ impl Completed {
             if let Some(doc) = doc {
                 extra.push(doc.to_owned());
             }
-            if let Some(types) = &annotation.types {
-                extra.push(types.label.tag.to_string());
+            if let Some(custom_contract_msg) = annotation
+                .types
+                .as_ref()
+                .and_then(|ty| ty.label.current_diagnostic())
+                .and_then(|diag| diag.message.as_ref())
+            {
+                extra.push(custom_contract_msg.clone());
```

Comment:
While we're here, is it really useful to display this in the LSP? 🤔

Reply:
I don't know? But at first sight, there is absolutely no reason (for now) that a label can have any custom diagnostic, as nothing is ever evaluated in the LSP. So I imagine we might get rid of that altogether